src/platform_impl/linux/mod.rs

```diff
@@ -904,7 +904,8 @@ impl ActiveEventLoop {
         x11_or_wayland!(match self; Self(evlp) => evlp.control_flow())
     }
 
-    pub(crate) fn clear_exit(&self) {
+    #[allow(dead_code)]
+    fn clear_exit(&self) {
         x11_or_wayland!(match self; Self(evlp) => evlp.clear_exit())
     }
```

Comment:
If it's dead, then just remove it.

Reply:
It's not dead, it's called by `self.window_target.p.clear_exit();` on Wayland, similar to `set_exit_code` and `exit_code`.

Reply:
I changed both platforms to do the same and removed `dead_code`.